gcc/rust/ast/rust-ast-visitor.cc

```diff
@@ -675,7 +675,46 @@ DefaultASTVisitor::visit (AST::AsyncBlockExpr &expr)
 
 void
 DefaultASTVisitor::visit (AST::InlineAsm &expr)
-{}
+{
+  visit_outer_attrs (expr);
+  using RegisterType = AST::InlineAsmOperand::RegisterType;
+  for (auto &operand : expr.get_operands ())
+    {
+      switch (operand.get_register_type ())
+	{
+	  case RegisterType::In: {
+	    operand.get_in ().expr->accept_vis (*this);
```

Comment:
Using `visit` directly would be more in line with the rest of the file and shorter but I don't mind merging this as is.